protocol/messenger_communities.go

```diff
@@ -1417,6 +1417,54 @@ func (m *Messenger) DeleteCommunityCategory(request *requests.DeleteCommunityCat
 	return &response, nil
 }
 
+func (m *Messenger) CommunityNewOwnershipKickAllAndRequestRejoin(communityID types.HexBytes) (*MessengerResponse, error) {
+	clock, _ := m.getLastClockWithRelatedChat()
+	changes, err := m.communitiesManager.PromoteSelfToControlNodeAndKickAllMembers(communityID, clock)
+	if err != nil {
+		return nil, err
+	}
+
+	err = m.communitiesManager.CommunityNewOwnershipGenerateRequestsToJoinForAutoApproval(changes.Community.ID(), changes.MembersRemoved)
```

Comment:
Possible new name
Basically, keeping the functions as verbs. Also, I don't think it's necessary to add `Community` in the name, since we're already in the `messenger_communities` file.
```suggestion
	err = m.communitiesManager.GenerateRequestsToJoinForAutoApprovalOnNewOwnership(changes.Community.ID(), changes.MembersRemoved)
```